### PR TITLE
Fix eslint plugin error

### PR DESCRIPTION
**Pull request by josiasds**
I did some debugging and find out the line that was failing the linter, so I just rewrote it.

### CODE REVIEW

**Comment by josiasds** [comment]
Merged on #166.

File: client/app/stores/commentsStore.js [hunk 1]
```diff
@@ -5,11 +5,10 @@ import reducers from '../reducers';
 import { initalStates } from '../reducers';
 
 export default props => {
-  const initialComments = props;
   const { $$commentsState } = initalStates;
   const initialState = {
     $$commentsStore: $$commentsState.merge({
-      $$comments: initialComments,
+      $$comments: props,
```
Comment (justin808) [inline client/app/stores/commentsStore.js]: This is bizarre.

1. Please back out this change. The old code is important for readability.
2. we should ignore the eslint rule here and with a comment to the github issue indicating the problem.
3. please put a link to this code in the eslint issue (actually react-plugin probably, not main eslint)

Thanks @josiasds

**Comment by justin808** [comment]
@josiasds @dylangrafmyre This needs to be reverted. See prior comments. Thanks.

File: client/app/stores/commentsStore.js (hunk 1 above)
Comment (dylangrafmyre) [inline client/app/stores/commentsStore.js]: @josiasds @justin808 Maybe you can just disable the rule for only this file or block? and not for the whole app. Don't know if the plugin will allow this?

**Comment by josiasds** [comment]
Previous change reverted and eslint rule disabled on commit bc076958012b176fa5aee1f5347e5153400b2426.

File: client/app/stores/commentsStore.js (hunk 1 above)
Comment (josiasds) [inline client/app/stores/commentsStore.js]: @dylangrafmyre yep, we just need to put `/* eslint react/prop-types: 0 */` at the top of the file, see bc076958012b176fa5aee1f5347e5153400b2426.

Comment (josiasds) [inline client/app/stores/commentsStore.js]: @justin808 Done :+1:

Comment (justin808) [inline client/app/stores/commentsStore.js]: :+1:

**Pull request closed without merging**